bundle/src/main/java/com/adobe/acs/commons/cloudservices/pwa/impl/PwaManifestServlet.java

```diff
@@ -111,21 +82,18 @@ private JsonObject getManifest(SlingHttpServletRequest request) throws ServletEx
         json.addProperty(KEY_DISPLAY,
                 manifest.get(PN_DISPLAY, "standalone"));
 
- 
         // TODO figure out the right getter
         json.addProperty(KEY_LANGUAGE,
```

Comment:
Is this really required ?  I did not find it in the docs https://developers.google.com/web/fundamentals/web-app-manifest/